app/models/spaCy/spacy.py

```diff
@@ -0,0 +1,19 @@
+from app.sen_py import SenPy
+import spacy
+
+class SpaCy(SenPy):
+  def __init__(self):
+    return None
+
+  def load(self):
+    nlp = spacy.load("./app/models/spaCy")
+
+    def predict(text):
+      return nlp(text).cats['POSITIVE']
+
+    nlp.predict = predict
+
+    return nlp
```

Comment:
origin of the scoring problem

Reply:
https://imgflip.com/s/meme/Jackie-Chan-WTF.jpg